ext/xsl/php_xsl.stub.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+class XSLTProcessor
+{
+    /** @return bool */
+    public function importStylesheet(object $stylesheet) {}
+
+    /** @return DOMDocument|false */
+    public function transformToDoc(DOMNode $document, ?string $return_class = null) {}
```

Comment:
This is `object` in zpp, and based on what you did below should maybe be `DOMDocument|SimpleXMLElement` as well?

Reply:
To be honest, I didn't exactly get this part of the code... However, I am ok with the suggested change. Can I add `object` as a param type and clarify it to `DOMDocument|SimpleXMLElement` via PHPDoc, right?